addons/storysource/README.md

```diff
@@ -11,27 +11,36 @@ This addon is used to show stories source in the addon panel.
 First, install the addon
 
 ```sh
-yarn add @storybook/addon-storysource --dev
+yarn add @storybook/addon-storysource @storybook/source-loader --dev
```

Comment:
source-loader is a dep of storysource, so this shouldn't be necessary

Reply:
Made the changes in a new commit